javascriptv3/example_code/nodegetstarted/README.md

```diff
@@ -1,34 +1,64 @@
+This guide shows you how to initialize an NPM package, add a service client to your package, and use the JavaScript SDK to call a service action.
 
-**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see 
-[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-example-javascript-syntax.html).
+## ⚠ Important
 
-# Getting started
+- Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/?aws-products-pricing.sort-by=item.additionalFields.productNameLowercase&aws-products-pricing.sort-order=asc&awsf.Free%20Tier%20Type=*all&awsf.tech-category=*all) and [Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all).
 
-1. Clone the [AWS SDK Code Samples repo](https://github.com/awsdocs/aws-doc-sdk-examples) to your local environment. See [the Github documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) for instructions.
+## Scenario
 
-2. Install the dependencies listed in the package.json.
+Create a new NPM package with one main file that does the following:
 
-**Note**: These include the client module for the AWS services required in these example, 
-which is *@aws-sdk/client-s3*.
-```
-npm install node -g
-cd javascriptv3/example_code/nodegetstarted
-npm install
-```
-3. In your text editor, update user variables specified in the ```Inputs``` section of the sample file.
+- Creates an Amazon Simple Storage Service (Amazon S3) bucket.
+- Puts an object in the bucket.
+- Reads the object.
+- Confirms if the user wants to delete resources.
 
-4. Run sample code:
-```
-cd src
-node [example name].js // For example, node sample.js
+### Prerequisites
+
+Before you can run the example, you must do the following:
+
+- Configure your [SDK authentication](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/getting-your-credentials.html)
+- [Install Node.js](https://nodejs.org/en/download)
+
+### Step 1: Set up the package structure
+
+1. Create a new folder to contain the package.
+2. From the command line, navigate to the new folder.
+3. Run `npm init -y`. This creates a default `package.json` file.
+4. Add `"type": "module"` to the `package.json`. This tells Node we're using modern ESM syntax.
+
+The final package.json should look similar to this:
+
+```json
+{
+  "name": "example-javascriptv3-get-started-node",
+  "version": "1.0.0",
+  "description": "This guide shows you how to initialize an NPM package, add a service client to your package, and use the JavaScript SDK to call a service action.",
+  "main": "index.js",
+  "scripts": {
+    "test": "vitest run **/*.unit.test.js"
+  },
+  "author": "Corey Pyle <corepyle@amazon.com>",
```

Comment:
Do you want your email in here?

Reply:
Always has been. Brian had his before mine. Haven't really thought about it.